pkg/services/dashboards/database/database_folder_test.go

```diff
@@ -170,7 +170,7 @@ func TestIntegrationDashboardFolderDataAccess(t *testing.T) {
 						FolderIds: []int64{
 							rootFolderId,
 							folder1.ID,
-						},
+						}, // nolint:staticcheck
```

Comment:
Will this work? I wonder if this should be attached to the line `FolderIds: []int64{`.

Reply:
it seems like it works since drone doesn't complain 🤔 , but good to note